### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by madisonestabrook** [fossasia/chat.susi.ai#1763]
**Actual Behaviour**
The project is not as accessible as it could be, especially to users who have disabilities: 
- Fix search area

**Expected Behaviour**
The project should be accessible to all users. 
**Steps to reproduce it**
Complete the following steps to reproduce the accessibly issues: 
1. Fork the repo and clone to your local machine
2. Use the` cd` command to move into the cloned folder in the Node.js command line 
3. Run `npm install` in the Node.js command line
4. Run 'npm start ' in the Node.js command line
5. Open Developer Tools
6. Inspect the code (example: the search button svg does not have alt text nor any ARIA).
HINT: select an element, click >>, and select Accessibility from the drop-down menu to view the selected element's accessibility proprieties.  

**Screenshots of the issue**
![image](https://user-images.githubusercontent.com/29613918/49038271-72e21480-f18a-11e8-86f7-3f53de636dcb.png)
Notice how accessibility can be improved. 
**Would you like to work on the issue?**
I would very much like to work on this issue!

**Comment by AakashMallik** [comment]
@madisonestabrook How do you propose to improve the accessibility of the app? As the issue, you raised falls under the category of enhancement so I guess a bit more elaboration is called for.

**Comment by madisonestabrook** [comment]
@AakashMallik  ARIA; however, I am willing to learn whatever tools are best for the job. I am currently studying for Google's Mobile Web Specialist Certification and recently graduated from the Grow with Google Front-End Web Development Nanodegree Scholarship program.

**Comment by AakashMallik** [comment]
@madisonestabrook Cool, go for it. Good luck. :smile:

**Comment by AakashMallik** [comment]
@madisonestabrook Any update on this issue?
